README.md

```diff
@@ -33,102 +33,471 @@ func main() {
+                            "height": 1080,
+                            "bitrate": 5535016.74
+                        }
+                    ],
+                    "duration": 734.083
+                },
+                "location": "s3://synq-frankfurt/videos/41/10/41101458-bc49-40db-badc-1b480831b79b/hls/master_manifest.m3u8",
+                "id": "de15f049-570b-465a-b5e3-96b3b0de4654",
+                "created_at": "2018-03-20T19:32:09.224649Z",
+                "account_id": null
+            },
+            {
+                "video_id": "41101458-bc49-40db-badc-1b480831b79b",
+                "url": "https://synq-player-zscvoibvg1af.stackpathdns.com/videos/41/10/41101458-bc49-40db-badc-1b480831b79b/smooth/signed_manifest.ism/manifest",
+                "updated_at": "2018-03-20T20:08:12.995477Z",
+                "type": "smooth",
+                "state": "completed",
+                "metadata": null,
+                "location": "s3://synq-frankfurt/videos/41/10/41101458-bc49-40db-badc-1b480831b79b/smooth/manifest.ism",
+                "id": "560a6ef0-9067-42c6-8e62-0f53f1c8bcdd",
+                "created_at": "2018-03-20T19:46:29.933076Z",
+                "account_id": null
+            },
+            {
+                "video_id": "41101458-bc49-40db-badc-1b480831b79b",
+                "url": "https://multicdn.synq.fm/videos/41/10/41101458-bc49-40db-badc-1b480831b79b/thumbnails/ff679a75-bbc6-4c70-814d-ea97ed8e43cf.jpg",
+                "updated_at": "2018-03-20T20:49:37.619508Z",
+                "type": "thumbnail",
+                "state": "completed",
+                "metadata": {},
+                "location": "https://multicdn.synq.fm/videos/41/10/41101458-bc49-40db-badc-1b480831b79b/thumbnails/ff679a75-bbc6-4c70-814d-ea97ed8e43cf.jpg",
+                "id": "ff679a75-bbc6-4c70-814d-ea97ed8e43cf",
+                "created_at": "2018-03-20T20:46:29.108781Z",
+                "account_id": null
+            }
+        ],
+        "account_ids": [
+            "1cf77745-dcdf-4ec1-8c83-cf92f0fb304d",
+            "716411e9-2101-4d12-a565-04a61c73980e",
+            "d2aa14aa-a32b-437c-ae34-7d47d95e28d9",
+            "95130d23-9893-4dd7-be9b-3ed742ec314b",
+            "83944a4f-1cb0-4e1b-bb03-70f1afc1d6a8",
+            "e4080e7b-3c0f-4981-9b80-4fb83e23777b",
+            "cf17ce25-eed4-4dd0-bd3e-95b820e92ada",
+            "874150c0-3032-4de5-b3e1-4852e30f7b7f",
+            "4c84d2e2-faed-480c-b447-99c52f0cd4c4",
+            "8c9c4189-825d-4d5b-9e47-11469fa14d75",
+            "db0de2e2-b812-4c99-a6e2-d9bf48b86e42",
+            "fcc90ae1-601a-4249-a562-ec4c1a52d53a"
+        ]
+    }
 }
 ```
 
+### Utilizing the testing framework
 
-## Usage (API v1)
-
-Here's an example of a simple main script that uses our SDK
+There's a pretty powerful mocked server in test_server/server.go which can be used for testing your service connected to the SDK.  Here's an example of how to use it
 
 ```golang
-package main
```

Comment:
Are you leaving this blank for now? Or did you forget to add a new example?

Reply:
Oh, actually, there was already v2 example, I just renamed it to just "Example"